stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetActivity.kt

```diff
@@ -214,6 +218,18 @@ internal class PaymentSheetActivity : BaseSheetActivity<PaymentSheetResult>() {
         }
     }
 
+    override fun onBackPressed() {
+        if (supportFragmentManager.backStackEntryCount > 0) {
+            updateErrorMessage(null)
+        }
+        super.onBackPressed()
+    }
```

Comment:
Does this need to be before `super.onBackPressed`? I think convention is call super first unless there's an explicit reason not to.

Reply:
Just used the same approach found here: https://git.corp.stripe.com/stripe-internal/android/pull/2955/files#r1231031

Reply:
I think after calling super the fragment will have been removed, so you won't know if there was any.